spec/syntax.md

```diff
@@ -352,14 +401,12 @@ Whitespace within a _pattern_ is meaningful and MUST be preserved.
 
 ### Expressions
 
-An **_expression_** is a _selector_ or a placeholder in a _pattern_
-that is evaluated during the formatting process. Each _expression_ MUST 
-start with an _operand_ or an _annotation_.
+An **_expression_** is a _selector_ or a _placeholder_ in a _pattern_
+that is evaluated during the formatting process.
```

Comment:
Sorry, should've noticed this earlier: This leaves out declarations as a possible place for expressions.